### DISCUSSION TITLE
Flash tuya touch switch

### DISCUSSION THREAD

**Discussion by diotta88** [indomus/forum#1935] [Hardware & Firmware]
Ciao a tutti. 
Sto cercando di flashare con tasmota il touch switch tuya di cui allego foto.
Ho provato con lo script tuya-convert, ma sembra che il fw è troppo aggiornato. 
Ho provato con il collegamento seriale, ma anche qui nulla da fare. Qualcuno lo ha già fatto? E ha qualche dritta da darmi? 

![image](https://user-images.githubusercontent.com/57493234/191373034-ef7d97b2-c41c-42f2-8de2-4cae5e74a8aa.jpeg)

![image](https://user-images.githubusercontent.com/57493234/191373096-b1889c63-e98e-4d32-9c66-9d6121fc9ea6.jpeg)

Grazie!

**Comment by snakuzzo** [comment]
Hai seguito tutte le indicazioni sulla [pagina dedicata](https://tasmota.github.io/docs/TuyaMCU-Devices/#additional-information) ?

**Comment by diotta88** [comment]
Si... ma non trovo dove e come mettere a massa RST del MCU. Altrimenti devo dissaldare RX sul tywe3s
